src/Tribe/Views/V2/View.php

```diff
@@ -2031,10 +2031,10 @@ protected function get_show_datepicker_submit() {
 	 */
 	protected function get_public_views( $url_event_date ) {
 		$public_views = tribe( Manager::class )->get_publicly_visible_views_data();
+		$query_args = wp_parse_url( $this->get_url(), PHP_URL_QUERY );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: 459b5a8df5ad24fc758e7a8f9c5a93ac</sub>